locales/de/yourloops.json

```diff
@@ -271,6 +271,7 @@
   "team-leave-dialog-question": "Möchten Sie dieses Behandlungsteam wirklich verlassen?",
   "team-leave-dialog-title": "Behandlungsteam verlassen",
   "team-member-admin": "Administrator",
+  "team-member-cannot-cancel-pending": "Nur der Einladende kann eine ausstehende Einladung stornieren.",
```

Comment:
You should not handle the translation yourself. All of this has to be done in crowdIn. You can revert all translated files expect the english one :laughing: